djng/forms/angular_base.py

```diff
@@ -132,7 +132,7 @@ def get_ng_widget_context(self, name, value, attrs):
     """
     Some widgets require a modified rendering context, if they contain angular directives.
     """
-    context = super(self.__class__, self).get_context(name, value, attrs)
+    context = self.__class__.get_context(self, name, value, attrs)
```

Comment:
in my opinion this would result into an infinite recursion, or wouldn't work at all.
``self.__class__.get_context(...)`` could only be called, if it would be a classmethod – or did I miss something here?